superset/bin/superset

```diff
@@ -4,7 +4,61 @@ from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
 
-from superset.cli import manager
+import argparse
+import sys
+
+import gunicorn.app.base
+
+
+class GunicornSupersetApplication(gunicorn.app.base.BaseApplication):
```

Comment:
Sure! The old version of `runserver` will still work until we deprecate it, so nothing should break immediately.